frontend/packages/console-app/src/components/cloud-shell/utils/cloudshell-resource.ts

```diff
@@ -0,0 +1,76 @@
+type environment = {
+  value: string;
+  name: string;
+};
+type Component = {
```

Comment:
`Component` is very generic name. It would be good idea to rename this to something else like `DevFileComponents`.